app/views/shared/_secondary_navigation.html.erb

```diff
@@ -188,6 +188,27 @@
               </a>
             </li>
           <% end %>
+          <% if can_view_repositories(@my_module.experiment.project.team) %>
```

Comment:
I would add `@my_module.experiment.project.team.repositories.exists?`, so when there are no custom repositories, the dropdown isn't shown, because it's empty in this case.